src/site/stages/build/index.js

```diff
@@ -38,6 +45,50 @@ const rewriteVaDomains = require('./plugins/rewrite-va-domains');
 const updateExternalLinks = require('./plugins/update-external-links');
 const updateRobots = require('./plugins/update-robots');
 
+/**
+ * Immediately copies the Webpack build output to a directory outside of
+ * Metalsmith's build destination, then returns a function for use as a
+ * Metalsmith plugin. This plugin copies the files back to their expected
+ * location.
+ *
+ * This is needed because script/build.sh runs Webpack before the content build,
+ * and Metalsmith's build method removes everything in the destination, which
+ * wipes out the output of the Webpack build.
+ *
+ * This can be removed when we move the content build to a new repository and
+ * this script no longer interacts with the Webpack output at all.
+ */
+function preserveWebpackOutput(metalsmithDestination) {
+  const webpackBuildDirName = 'generated';
+  const tempDir = path.join(__dirname, '../../../../tmp/', webpackBuildDirName);
```

Comment:
Where does this `tmp` directory come from?

Reply:
Right here. :smile: 

Like I explained in the comment for this function, `smith.build()` nukes _everything_ in the `build/${buildtype}/`, so this function
1. Moves the output from the Webpack build into a temporary folder outside of `build/` so we don't lose it
1. Returns a plugin that moves it back during the execution of the pipeline, after `.build()` clears the directory out